frontend/lib/evictionfree/homepage.tsx

```diff
@@ -20,8 +21,13 @@ export const JUSTFIX_WEBSITE_URLS = {
 
 type EvictionFreeImageType = "png" | "svg" | "jpg";
 
-export function getEFImageSrc(name: string, type?: EvictionFreeImageType) {
-  return `frontend/img/evictionfree/${name}.${type || "svg"}`;
+export function getEFImageSrc(
+  name: string,
+  type?: EvictionFreeImageType,
+  islocalized?: boolean
+) {
+  const fileName = islocalized ? `${name}_${li18n.language}` : name;
```

Comment:
@toolness is there any way to assert that a file with path `${name}_${li18n.language}` exists?

Reply:
Sounds like a no